CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v1.6.1-1] - 2021-11-30
 
 ### Changed
-- use word-break and hyphens in toc and pageviewer so that long words get wrapped #169 
+- Use word-break and hyphens in toc and pageviewer so that long words get wrapped #169
+- Update images in UI guide documentation 
```

Comment:
this belongs in the unreleased section and misses the edit of the menu

Reply:
- [x] done